src/vm/datatypes/strings.c

```diff
@@ -239,7 +270,13 @@ static Value findString(DictuVM *vm, int argCount, Value *args) {
         runtimeError(vm, "find() takes either 1 or 2 arguments (%d given)", argCount);
         return EMPTY_VAL;
     }
-
+    {
```

Comment:
Do we need the extra scope here? It seems to happen throughout quite a few times

Reply:
I have refactored them to not do that anymore, it was for name shadowing but the fix was pretty easy